generators/directive/index.js

```diff
@@ -1,14 +1,19 @@
 'use strict';
 
-var Generator = require('yeoman-generator'),
-    chalk     = require('chalk'),
-    yosay     = require('yosay'),
-    fs        = require('fs');
+/**
+ * Generates a directive.
+ */
+
+var Generator  = require('yeoman-generator'),
+    chalk      = require('chalk'),
+    yosay      = require('yosay'),
+    caseIt     = require('change-case'),
+    fsp        = require('fs-promise');
```

Comment:
this wraps the `fs` module functionality with promise-based functionality, which makes our code more elegant and readable.